### PR TITLE
Connecting to mongdb server.

### PR DESCRIPTION
**Pull request by Flaagrah**
Connecting to the database

### CODE REVIEW

**Review by pavel6767 — APPROVED** [review]
looks good!

I'm assuming /server/config/default.json will be updated and git-ignored once we type in our own users and passwords

**Comment by Flaagrah** [comment]
Using .env file for db url rather than default.json

File: server/config/default.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (aeciorc) [inline server/config/default.json:1]: For the .env equivalent of a default.json, you can use  .env.template

File: server/db.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
```
Comment (aeciorc) [inline server/db.js:2]: This is redundant if I'm not mistaken, require('dotenv').config() should only need to be performed once.

**Pull request merged**